prefect_saturn/core.py

```diff
@@ -104,7 +104,7 @@ def _hash_flow(self, flow: Flow) -> str:
         hasher.update(cloudpickle.dumps(identifying_content))
         return hasher.hexdigest()
 
-    def _set_flow_metadata(self, flow: Flow) -> None:
+    def _set_flow_metadata(self, flow: Flow, instance_size: Optional[str] = None) -> None:
```

Comment:
sorry, one more thing I just thought of. Can you please make `instance_size` required here?

In my experience, when you use this pattern of "public methods take in arguments that they pass through to private methods", it's easy to make mistakes in the upstream code and not correctly pass things through. One way to catch those mistakes is to never allow private methods used in such a pattern to take default values.

```suggestion
    def _set_flow_metadata(self, flow: Flow, instance_size: Union[str, None]) -> None:
```

Reply:
Good call